tests/unit/OpenFin/openfin.spec.ts

```diff
@@ -192,6 +192,12 @@ describe("OpenFinContainer", () => {
         expect(container.hostType).toEqual("OpenFin");
     });
 
+    it("getMainWindow returns wrapped inner window", () => {
```

Comment:
Since I was able to get the mock Window static working for the new test I added the always missing coverage of getMainWindow